py3status/modules/volume_status.py

```diff
@@ -250,21 +292,42 @@ def deprecate_function(config):
             ]
         }
 
+    def __init__(self):
+        self._format_specified = False
+        self._format_muted_specified = False
```

Comment:
these can be set in `post_config_hook()`.

`__init__()` is discouraged as the module is not fully initialized at this point.

I realise what you are trying to do but we can do this more cleanly, no ` __setattr__()` etc.  For now I think we should get the functionality added first and then worry about the format.